src/routes/teams.js

```diff
@@ -926,6 +926,20 @@ async function inviteTeamMember(request, h) {
         invite_token: token
     };
 
+    if (payload.role === ROLES[0]) {
```

Comment:
I'd just hard-code the values here to make them easier to read
```suggestion
    if (payload.role === 'owner') {
```